UnitsNet/UnitConverter.cs

```diff
@@ -99,17 +85,17 @@ public static bool TryConvert(QuantityValue fromValue, Enum fromUnitValue, Enum
         /// <exception cref="AmbiguousUnitParseException">More than one unit matches the abbreviation.</exception>
         public static double ConvertByName(QuantityValue fromValue, string quantityName, string fromUnit, string toUnit)
         {
-            if (!TryGetUnitType(quantityName, out Type unitType))
+            if (!TryGetUnitType(quantityName, out var unitType))
                 throw new UnitNotFoundException($"The unit type for the given quantity was not found: {quantityName}");
 
-            if (!TryParseUnit(unitType, fromUnit, out Enum fromUnitValue)) // ex: LengthUnit.Meter
+            if (!TryParseUnit(unitType, fromUnit, out var fromUnitValue)) // ex: LengthUnit.Meter
```

Comment:
Here too, in my mind it's not super obvious that the type is `Enum` although the comment helps hint at it and you can reason about it. I just prefer it to not have to think so much when reading.

Reply:
See above.

Reply:
See my counter-argument above 😆 Here too, I wouldn't know if it was `object`, `Enum` or perhaps even a specific enum type if I don't know the code well.

Reply:
Done!